### PR TITLE
Fix group argument of {de,}convolution_2d to keyword-only argument. 

### PR DESCRIPTION
**Pull request by okuta**
I took over #4059. This PR contains fixing about incompatible interface bug.

### CODE REVIEW

File: chainer/links/connection/convolution_2d.py [hunk 1]
```diff
@@ -7,14 +7,15 @@
...
-    """__init__(self, in_channels, out_channels, ksize=None, stride=1, pad=0, nobias=False, initialW=None, initial_bias=None)
+    """__init__(self, in_channels, out_channels, ksize=None, stride=1, pad=0, nobias=False, initialW=None, initial_bias=None, *, groups=1)
 
     Two-dimensional convolutional layer.
 
     This link wraps the :func:`~chainer.functions.convolution_2d` function and
     holds the filter weight and bias vector as parameters.
 
-    The output of this function can be non-deterministic when it uses cuDNN.
+    The output of this functio
```
Comment (kmaehashi) [inline chainer/links/connection/convolution_2d.py]: This line seems broken.

Comment (okuta) [reply chainer/links/connection/convolution_2d.py]: Thanks, I will fix.

**Comment by kmaehashi** [comment]
LGTM!

**Comment by okuta** [comment]
Thanks.

**Pull request merged**